core/constants.go

```diff
@@ -397,6 +397,73 @@ const AsyncCallStepField = "AsyncCallStep"
 // before executing the destination async call, to be put aside for the async callback
 const AsyncCallbackGasLockField = "AsyncCallbackGasLock"
 
+const (
+	// MetricScDeployEpoch represents the epoch when the deployment of smart contracts will be enabled
+	MetricScDeployEpoch = "smart_contract_deploy"
```

Comment:
I would recommend the suffix `EnableEpoch` instead of `Epoch`

Reply:
also all the metrics should have the prefix `erd_`. So:

`erd_smart_contract_deploy_enable_epoch`
apply to all the metrics

Reply:
done